packages/js/src/Modules/Verto/webrtc/helpers.ts

```diff
@@ -29,7 +29,7 @@ const _constraintsByKind = (
   kind: string = null
 ): { audio: boolean; video: boolean } => {
   return {
-    audio: !kind || kind === DeviceType.AudioIn,
+    audio: !kind || kind === DeviceType.AudioIn || kind === DeviceType.AudioOut,
```

Comment:
Why are we using `DeviceType.AudioOut`?

Reply:
The SDK is using `getAudioOutDevices` here, to be able to list the Audio Out Put Devices, for do that we need to add the permission of {audio: true}

Reply:
https://github.com/team-telnyx/webrtc/pull/203/files#diff-2447a25963f6b87422abbb0c3432ebcd9ebaf2c4e9abef301057d962b60a45b6R327

Reply:
Okay, just wanted to be sure this was intentional.